common/config/api.go

```diff
@@ -143,6 +143,8 @@ func NewContextAPIConfig(c *cli.Context) *ContextAPIConfig {
 }
 
 func (c *ContextAPIConfig) Validate() error {
+	flags := APIFlags()
+
 	requiredVars := []string{
```

Comment:
why not just convert this from `[]string` to `map[string]string` where the val is the corresponding environment variable?

Reply:
✅